pkgs/applications/editors/emacs/elisp-packages/update-melpa

```diff
@@ -1,6 +1,6 @@
 #! /usr/bin/env nix-shell
 #! nix-shell --show-trace ./updater-emacs.nix -i bash
 
-SCRIPT_DIR="$( cd "$(dirname "$0")" ; pwd -P )"
+SCRIPT_DIRECTORY=$(cd "$(dirname "${BASH_SOURCE[@]}")"; cd -P "$(dirname "$(readlink "${BASH_SOURCE[@]}" || echo .)")"; pwd)
 
-exec emacs --fg-daemon=updater --quick -l $SCRIPT_DIR/update-melpa.el -f run-updater "$@"
+exec emacs --fg-daemon=updater --quick -l $SCRIPT_DIRECTORY/update-melpa.el -f run-updater "$@"
```

Comment:
Out of an abundance of caution, I’d suggest wrapping `$SCRIPT_DIRECTORY` in double quotes (as in previous instances of this commit), to protect against spaces in paths, which would break the command.
Either `"$SCRIPT_DIRECTORY"/update-melpa.el` or `"$SCRIPT_DIRECTORY/update-melpa.el"`, depending on your taste. (I’m partial to the first form, but not sure what the custom is in nixpkgs, if any).

Reply:
(and maybe, for consistency, use also the `"${SCRIPT_DIRECTORY}"` form here as well).

Reply:
Well remembered. I will fix it rn.